client/src/App.js

```diff
@@ -224,9 +225,13 @@ render() {
   return (
       <Sidebar
         sidebar={<>
-          <h2 className="SideLink"><Link to="/" style={linkStyle}>Home</Link></h2>
-          <h2 className="SideLink"><Link to="/about/" style={linkStyle}>About</Link></h2>
-          <h2 className="SideLink"><Link to="/materials/" style={linkStyle}>Materials</Link></h2>
+          <Link to="/" style={linkStyle} 
+            onClick={this.onSetSidebarOpen}><h2 className="SideLink">Home</h2></Link>
```

Comment:
when clicking a link, `onSetSidebarOpen` is not required to jump to the page.  but, we should close the sidebar and also the burger icon back to normal. So, probably the function name change to `toggleSideBar` or something and the function should be toggling one state instead of using two states. 

Let's consider this improvement in another PR.